src/patternfly/components/Alert/alert.scss

```diff
@@ -3,215 +3,174 @@
-    --pf-c-alert--m-inline--before--BackgroundColor: var(--pf-c-alert--m-inline--m-warning--before--BackgroundColor);
-
-    // Apply a custom size to the FA icon to ensure all icons are the same width
-    .pf-c-alert__icon > i {
-      font-size: var(--pf-c-alert--m-warning__icon--FontSize);
-    }
-
-    // Apply a custom size to the FA icon to ensure all icons are the same width
-    &.pf-m-inline .pf-c-alert__icon > i {
-      font-size: var(--pf-c-alert--m-inline--m-warning__icon--FontSize);
-    }
-
-    &.pf-m-inline {
-      --pf-c-alert__icon--Color: var(--pf-c-alert--m-inline--m-warning__icon--Color);
-    }
+    --pf-c-alert--m-inline--BackgroundColor: var(--pf-c-alert--m-inline--m-warning--BackgroundColor);
   }
 
   &.pf-m-info {
+    --pf-c-alert--before--BorderTopColor: var(--pf-c-alert--m-info--before--BorderTopColor);
     --pf-c-alert__icon--Color: var(--pf-c-alert--m-info__icon--Color);
-    --pf-c-alert__icon--BackgroundColor: var(--pf-c-alert--m-info__icon--BackgroundColor);
     --pf-c-alert__title--Color: var(--pf-c-alert--m-info__title--Color);
-
-    // Set border left color in case it is also an inline alert
-    --pf-c-alert--m-inline--before--BackgroundColor: var(--pf-c-alert--m-inline--m-info--before--BackgroundColor);
-
-    &.pf-m-inline {
-      --pf-c-alert__icon--Color: var(--pf-c-alert--m-inline--m-info__icon--Color);
-    }
+    --pf-c-alert--m-inline--BackgroundColor: var(--pf-c-alert--m-inline--m-info--BackgroundColor);
   }
 
   &.pf-m-inline {
     --pf-c-alert--BoxShadow: var(--pf-c-alert--m-inline--BoxShadow);
-    --pf-c-alert__icon--BackgroundColor: var(--pf-c-alert--m-inline__icon--BackgroundColor);
-    --pf-c-alert__icon--Color: var(--pf-c-alert--m-inline__icon--Color);
-
-    border: solid var(--pf-c-alert--m-inline--BorderColor);
-    border-width: var(--pf-c-alert--m-inline--BorderTopWidth) var(--pf-c-alert--m-inline--BorderRightWidth) var(--pf-c-alert--m-inline--BorderBottomWidth) var(--pf-c-alert--m-inline--BorderLeftWidth);
-
-    // Left border treatment
-    &::before {
-      position: absolute;
-      top: var(--pf-c-alert--m-inline--before--Top);
-      bottom: var(--pf-c-alert--m-inline--before--Bottom);
-      left: 0;
-      width: var(--pf-c-alert--m-inline--before--Width);
-      content: "";
-      background-color: var(--pf-c-alert--m-inline--before--BackgroundColor);
-    }
-
-    .pf-c-alert__icon {
-      --pf-c-alert__icon--FontSize: var(--pf-c-alert--m-inline__icon--FontSize);
-
-      padding: var(--pf-c-alert--m-inline__icon--PaddingTop) var(--pf-c-alert--m-inline__icon--PaddingRight) var(--pf-c-alert--m-inline__icon--PaddingBottom) var(--pf-c-alert--m-inline__icon--PaddingLeft);
-    }
+    --pf-c-alert--BackgroundColor: var(--pf-c-alert--m-inline--BackgroundColor);
   }
 }
 
 .pf-c-alert__icon {
   display: flex;
-  grid-area: icon;
-  padding: var(--pf-c-alert__icon--Padding);
+  grid-row: 1 / span 1;
+  grid-column: 1 / span 1;
```

Comment:
Is there a reason you don't want it to use the `icon` grid area?` Since it's defined and the rest of the sections use `grid-area`, you could modify the template areas and use `.` for the 2nd and 3rd column `icon` areas currently.

Reply:
oh good point didn't realize you can do `.` nice!